gp-includes/routes/translation.php

```diff
@@ -265,11 +265,13 @@ public function translations_post( $project_path, $locale_slug, $translation_set
 					$data[ "translation_$i" ] = $translations[ $i ];
 				}
 			}
+			$set_status = $data['status'];
+			$data['status'] = 'waiting';
```

Comment:
This change was necessary because if the translations ends up being set to `waiting` because of Cross-PTE then two `current` translations would exist.